studio/src/pages/claims/index.js

```diff
@@ -115,7 +147,34 @@ function Claims({ permission }) {
     });
   };
 
-  return loading ? (
+  if (!loading && !claimantsLoading && claimantsCount === 0){
+    return (
+    <Result
+    status={'403'}
+    title={'Please create a claimant.'}
+    subTitle="You cannot create a claim without a claimant."
+    extra={
+      <Link to="/claimants/create">
+        <Button type="primary">Create claimant</Button>
+      </Link>
+    }
+  />)
+  }
+
```

Comment:
@ishpaul777 can the case when both ratings and claimants aren't present be handled?

Reply:
we can refactor this to have create rating and create claimants button inside and then conditionally render both or one.